src/east/system_commands/tooling.py

```diff
@@ -281,7 +285,7 @@ def _install_codechecker(east: EastContext, exe_path: str):
 [bold green]codechecker install done![/]
 """
 
-packages = [
+supported_tools = [
```

Comment:
Maybe remove conda from the list?

Reply:
Yes, I will. I don't think that we will need it in the near future.